.github/workflows/release.yml

```diff
@@ -30,7 +30,9 @@ jobs:
 
       - name: auto-dist-tag
         run: npx auto-dist-tag@1 --write
```

Comment:
Note: seems weird we use npx here

Reply:
that's how it is here: https://github.com/mainmatter/qunit-dom/blob/master/.github/workflows/release.yml#L32, on main

Reply:
Yeah :+1: commented only as a note to myself.